main/src/ca/uwaterloo/flix/language/ast/TokenKind.scala

```diff
@@ -91,12 +95,18 @@ object TokenKind {
 
   case object Dot extends TokenKind
 
+  case object DotCurlyL extends TokenKind
```

Comment:
What does DotCurlyL do?

Reply:
`.{` is used in use-expressions `use Color.{green, red}`. Tokenizing it to a single token rather than `Dot, CurlyL` simplyfies parsing a little since there is less overlap with qualified names
```
// qualified name ending in use of multiple symbols
MyLib.Color.{green} -> UppercaseName Dot UppercaseName DotCurlyL LowercaseName CurlyR
```

Reply:
Ok, thanks.